Error while checking build/O2/fullCI for d00326f1dc5b129f1b494d3622cb30cddf1cf665 at 2023-05-17 12:18:
```
## sw/BUILD/O2-full-system-test-latest/log
command /sw/slc8_x86-64/O2/11327-slc8_x86-64-local1/prodtests/full-system-test/dpl-workflow.sh had nonzero exit code 1
[ERROR] invalid workflow in o2-fv0-integrate-cluster-workflow: No matching output found for FV0/RECPOINTS/0 as requested by data processor "fv0-integrate-clusters". Candidates:
[ERROR] -DPL/ENUM/1301969186
[ERROR] -CPV/RAWDATA
[ERROR] -CTP/RAWDATA
[ERROR] -EMC/RAWDATA
[ERROR] -FDD/RAWDATA
[ERROR] -FLP/DISTSUBTIMEFRAME/0
[ERROR] -FLP/DISTSUBTIMEFRAME/52443
[ERROR] -FT0/RAWDATA
[ERROR] -FV0/RAWDATA
[ERROR] -HMP/RAWDATA
[ERROR] -ITS/RAWDATA
[ERROR] -MCH/RAWDATA
[ERROR] -MFT/RAWDATA
[ERROR] -MID/RAWDATA
[ERROR] -PHS/RAWDATA
[ERROR] -TOF/RAWDATA
[ERROR] -TPC/RAWDATA
[ERROR] -TRD/RAWDATA
[ERROR] -ZDC/RAWDATA
[ERROR] -FLP/DISTSUBTIMEFRAME/1
[ERROR] -FLP/DISTSUBTIMEFRAME/2
[ERROR] -FLP/DISTSUBTIMEFRAME/3
[ERROR] -FLP/DISTSUBTIMEFRAME/4
[ERROR] -FLP/DISTSUBTIMEFRAME/5
[ERROR] -FLP/DISTSUBTIMEFRAME/6
[ERROR] -FLP/DISTSUBTIMEFRAME/7
[ERROR] -FLP/DISTSUBTIMEFRAME/8
[ERROR] -FLP/DISTSUBTIMEFRAME/9
[ERROR] -FLP/DISTSUBTIMEFRAME/10
[ERROR] -FLP/DISTSUBTIMEFRAME/11
[ERROR] -FLP/DISTSUBTIMEFRAME/12
[ERROR] -FLP/DISTSUBTIMEFRAME/13
[ERROR] -FLP/DISTSUBTIMEFRAME/14
[ERROR] -FLP/DISTSUBTIMEFRAME/15
[ERROR] -CPV/CPV_BadMap/0
[ERROR] -CPV/CPV_Gains/0
[ERROR] -CPV/CPV_Pedestals/0
[ERROR] -CPV/CPV_SimPars/0
[ERROR] -CPV/CTFDICT/0
[ERROR] -CTP/CTFDICT/0
[ERROR] -CTP/ORBITRESET/0
[ERROR] -CTP/OrbitReset/0
[ERROR] -CTP/Trig_Offset/0
[ERROR] -EMC/CTFDICT/0
[ERROR] -EMC/RECOPARAM/0
[ERROR] -FDD/CTFDICT/0
[0 more errors; see full log]
```
Full log [here](https://ali-ci.cern.ch/alice-build-logs/AliceO2Group/AliceO2/11327/d00326f1dc5b129f1b494d3622cb30cddf1cf665/build_O2_fullCI/pretty.html).

--- a/prodtests/full-system-test/calib-workflow.sh
+++ b/prodtests/full-system-test/calib-workflow.sh
@@ -16,6 +16,12 @@ if workflow_has_parameters CALIB_LOCAL_INTEGRATED_AGGREGATOR CALIB_PROXIES; then
   exit 2
 fi
 
+if [[ -z ${TPC_IDCCALIB_NSLICESPERTF:-} ]]; then
+  TPC_IDCCALIB_NSLICESPERTF=$(echo "scale=2;$NHBPERTF/128*11" | bc)
+  TPC_IDCCALIB_NSLICESPERTF=$(echo $TPC_IDCCALIB_NSLICESPERTF | awk '{print int($1 + 0.5)}')
+fi
+
+
 if [[ "${CALIB_TPC_SCDCALIB_SENDTRKDATA:-}" == "1" ]]; then ENABLE_TRKDATA_OUTPUT="--send-track-data"; else ENABLE_TRKDATA_OUTPUT=""; fi
 
 # specific calibration workflows
@@ -33,10 +39,10 @@ if [[ $CALIB_ZDC_TDC == 1 ]]; then add_W o2-zdc-tdccalib-epn-workflow "" "" 0; f
 if [[ $CALIB_FT0_TIMEOFFSET == 1 ]]; then add_W o2-calibration-ft0-time-spectra-processor; fi
 
 # current integration
-if [[ $CALIB_FT0_INTEGRATEDCURR == 1 ]]; then add_W o2-ft0-integrate-cluster-workflow "$DISABLE_ROOT_OUTPUT"; fi
-if [[ $CALIB_FV0_INTEGRATEDCURR == 1 ]]; then add_W o2-fv0-integrate-cluster-workflow "$DISABLE_ROOT_OUTPUT"; fi
-if [[ $CALIB_FDD_INTEGRATEDCURR == 1 ]]; then add_W o2-fdd-integrate-cluster-workflow "$DISABLE_ROOT_OUTPUT"; fi
-if [[ $CALIB_TOF_INTEGRATEDCURR == 1 ]]; then add_W o2-tof-integrate-cluster-workflow "$DISABLE_ROOT_OUTPUT"; fi
+if [[ $CALIB_FT0_INTEGRATEDCURR == 1 ]]; then add_W o2-ft0-integrate-cluster-workflow "$DISABLE_ROOT_OUTPUT --nSlicesTF $TPC_IDCCALIB_NSLICESPERTF"; fi
+if [[ $CALIB_FV0_INTEGRATEDCURR == 1 ]]; then add_W o2-fv0-integrate-cluster-workflow "$DISABLE_ROOT_OUTPUT --nSlicesTF $TPC_IDCCALIB_NSLICESPERTF"; fi
+if [[ $CALIB_FDD_INTEGRATEDCURR == 1 ]]; then add_W o2-fdd-integrate-cluster-workflow "$DISABLE_ROOT_OUTPUT --nSlicesTF $TPC_IDCCALIB_NSLICESPERTF"; fi
+if [[ $CALIB_TOF_INTEGRATEDCURR == 1 ]]; then add_W o2-tof-integrate-cluster-workflow "$DISABLE_ROOT_OUTPUT --nSlicesTF $TPC_IDCCALIB_NSLICESPERTF"; fi
 
 # for async calibrations
 if [[ $CALIB_EMC_ASYNC_RECALIB == 1 ]]; then add_W o2-emcal-emc-offline-calib-workflow "--input-subspec 1 --applyGainCalib"; fi

